src/chromeDebugAdapter.ts

```diff
@@ -130,13 +130,26 @@ export class ChromeDebugAdapter extends CoreDebugAdapter {
         return super.attach(args);
     }
 
-    public configurationDone(): Promise<void> {
+    protected hookConnectionEvents(): void {
+        super.hookConnectionEvents();
+        this.chrome.Page.onFrameNavigated(params => this.onFrameNavigated(params));
+    }
+
+    protected onFrameNavigated(params: Crdp.Page.FrameNavigatedEvent): void {
+        if (params.frame.url === this._userRequestedUrl) {
+            // Chrome started to navigate to the user's requested url
+            this.events.emit(ChromeDebugSession.FinishedStartingUpEventName);
+        }
+    }
+
+    public async configurationDone(): Promise<void> {
         if (this._userRequestedUrl) {
             // This means all the setBreakpoints requests have been completed. So we can navigate to the original file/url.
             this.chrome.Page.navigate({ url: this._userRequestedUrl });
+            this.events.emitMilestoneReached("RequestedNavigateToUserPage");
```

Comment:
Wait on the navigate before sending the event? Maybe don't await so it doesn't block configurationDone. But just like `Page.navigate(...).then(() => emitMilestoneReached())`